src/components/renderers/TechnologiesRenderer.tsx

```diff
@@ -0,0 +1,31 @@
+import { ProfileModel } from '@/data/frontend/profile/types'
+
```

Comment:
**issue:** We should add `use client` here, as it uses `window` - this renderer will crash when rendered on server